CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.11.3
+
+* Fix `U64MarkerAllocator`'s internal index not being updated on `saveload::DeserializeComponents`
```

Comment:
You forgot to add the link here ;)

Reply:
Oh and please add the `LazyUpdate` change, too.

Reply:
Didn't notice those, thanks! Forgot to add .marked() as well.

Commit amended.